FXyz-Core/src/main/java/org/fxyz3d/scene/Skybox.java

```diff
@@ -121,7 +121,13 @@ public Skybox(Image topImg, Image bottomImg, Image leftImg, Image rightImg, Imag
         startTimer();
     }
 
-    
+    public void setEnableTimer(boolean enable) {

```

Comment:
I take this is just new API to allow developers access the timer? Developers can use it to start/stop the timer, but there is no internal handling of this?

Do you think the samples that use the skyBox could benefit from this API?

For instance, would make sense do have a listener to the scene property of the skyBox, so whenever it is part of the scene, timer starts, and when removed it stops?

We don't have javadoc (yet), but maybe we can add here:
```
/**
 * @since 0.5.5
 */
```?

Reply:
Yes only intent is to give Developers full control. Crybaby in #133 seemed to have a use case for rapidly adding and removing Skybox's from the scene.

I'm on the fence for adding a listener since it adds behavior that is hidden by default. I suppose a compromise would be adding a bindable property which gives the developer an option via binding to automatically start/stop? 

Ahh javadoc... you're going to make me do this the right way aren't you? :-)

Reply:
Ok so for updating the samples... you're suggesting updating the SkyBox sample with some control that uses the new public API call right?

Reply:
I think we need a follow up on any potential sample changes simply because I dont see what controls we could add currently.